### ISSUE TITLE
YouTube endpoint for video information has been discontinued

### ISSUE THREAD

**Issue by abjerner** [warrenbuckley/UmbracoExtensionMethods#28]
The URL used in the code no longer works - se example on the URL below:

https://github.com/warrenbuckley/UmbracoExtensionMethods/blob/master/ExtensionMethods/YouTube/YouTubeVideo.cs#L35

I'm looking into whether there is another way obtaining the same information.

**Comment by warrenbuckley** [comment]
Not easily with public API endpoint @abjerner I believe. As the OLD V2 API got deprecated

**Comment by abjerner** [comment]
Yup. Looking into this at the moment since we have a client where their video feed is down. V2 has been deprecated for a while, but not it seems it has been removed.

There is an equivalent in V3, but calling it without an access token or server key will almost always return an error like `Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.`. So it seems that there is no way anymore to accomplish this without first registering an app in the Developer Console, and then continue from there.
